modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/clay/_components.scss

```diff
@@ -60,10 +77,10 @@
 @import 'components/_timelines';
 @import 'components/_toggle-switch';
 @import 'components/_tooltip';
-@import 'components/_type';
 
 @import 'components/_utilities';
+@import 'components/_utilities-functional-important';
 
 @import 'components/_loaders';
 
-@import '../compat/components';
```

Comment:
You're probably right, @eduardoallegrini

Reply:
![whew](https://user-images.githubusercontent.com/788266/87063179-d9327400-c1c2-11ea-8828-6303d03bd2a6.gif)